pepper-apis/dsm-core/src/main/java/org/broadinstitute/dsm/util/proxy/jackson/ObjectMapperSingleton.java

```diff
@@ -0,0 +1,45 @@
+package org.broadinstitute.dsm.util.proxy.jackson;
+
+import java.util.Map;
+import java.util.Objects;
+
+import com.fasterxml.jackson.core.type.TypeReference;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import org.apache.commons.lang3.StringUtils;
+
+public class ObjectMapperSingleton {
+
+    private ObjectMapperSingleton() {
+    }
+
+    public static ObjectMapper instance() {
+        return Helper.objectMapperInstance;
+    }
+
+    public static <T> T readValue(String content, TypeReference<?> typeReference) {
+        content = StringUtils.isBlank(content) ? "{}" : content;
+        return null;
+//        try {
+//            return Helper.objectMapperInstance.readValue(content, typeReference);
+//        } catch (com.fasterxml.jackson.core.JsonParseException e) {
+//            throw new JsonParseException(e.getMessage());
+//        } catch (IOException e) {
+//            throw new RuntimeException(e);
+//        }
```

Comment:
@gmakharat @Anzori-Ghurtchumelia can you please have a look at that? I needed to comment it out to get it compiling